Update changelog with a note about the addition of new log profiles (#2066)
nginx/documentation: content/nginx-one-console/changelog.md

Stay up-to-date with what's new and improved in the F5 NGINX One Console.

## June 23, 2026

### F5 WAF for NGINX: Added additional built-in log profiles

> Nine additional built-in WAF log profiles are now available out of the box, covering common logging formats (default, gRPC, ArcSight, Splunk).

Reviewer: ```suggestion
Nine additional built-in WAF log profiles are now available, covering common logging formats: default, gRPC, ArcSight, and Splunk.
```

Removed idiomatic language "out of the box" -- might not be understood by our global audience.
